Sources/Network.swift

```diff
@@ -132,9 +132,9 @@ class Network {
                              properties: Properties = [:],
                              completion: ((Bool) -> Void)? = nil) {
         let trackProperties = properties.merging(["token": apiToken,
-                                                  "mp_lib": properties["mp_lib"] ?? "swift",
+                                                  "mp_lib": "swift",
                                                   "distinct_id": distinctId,
-                                                  "$lib_version": properties["$lib_version"] ?? AutomaticProperties.libVersion()]) {(current, _) in current }
+                                                  "$lib_version": AutomaticProperties.libVersion()]) {(current, _) in current }
```

Comment:
The `{(current, _) in current }` closure keeps the values in `properties` for any keys that are already present.

Reply:
https://developer.apple.com/documentation/swift/dictionary/3127175-merging